sql/moz-fx-data-shared-prod/firefox_desktop_background_defaultagent_derived/default_agent_agg_v1/query.sql

```diff
@@ -0,0 +1,21 @@
+SELECT
+  DATE(submission_timestamp) AS submission_date,
+  mozfun.norm.truncate_version(client_info.app_display_version, 'major') AS major_build_version,
+  normalized_channel,
+  metrics.string.system_default_browser,
+  metrics.string.system_default_previous_browser,
+  metrics.boolean.notification_show_success,
+  metrics.string.notification_action,
+  COUNT(*) AS row_count
```

Comment:
We also need the OS version on this table.
e.g.: https://sql.telemetry.mozilla.org/queries/74902/source

Reply:
nevermind, I think I mixed up the tables